### ISSUE TITLE
Warning: Received `false` for a non-boolean attribute `autoCorrect`.

### ISSUE THREAD

**Issue by jon301** [ianstormtaylor/slate#4248]
**Description**
There are some React warnings in a test environment (jsdom with Jest) regarding `autoCorrect` and `autoCapitalize` attributes.

**Recording**
![image](https://user-images.githubusercontent.com/122108/117320102-33a46b80-ae8c-11eb-9cb8-ed6e8e6269cd.png)

**Environment**
- Slate Version: 0.62.1-2021326135112
- Operating System: macOS 11.3.1
- TypeScript Version: 4.1.2

**Context**
After digging a bit, it seems like the warning is thrown because of this [line](https://github.com/ianstormtaylor/slate/blob/623960a7d14103b8cebe667019b4de5f8ad1fd61/packages/slate-react/src/components/editable.tsx#L521) which assigns the boolean `false` to `autoCorrect`  (whereas it expects a string) in case `HAS_BEFORE_INPUT_SUPPORT` is `false`, which is the case in a jsdom environment.

**Comment by craigdallimore** [comment]
The same applies to `autoCapitalize`, if I understand correctly. `jest` will also error on that:
```
Warning: Received `false` for a non-boolean attribute `autoCapitalize`.
      
      If you want to write it to the DOM, pass a string instead: autoCapitalize="false" or autoCapitalize={value.toString()}.
      
      If you used to conditionally omit it with autoCapitalize={condition && value}, pass autoCapitalize={condition ? value : undefined} instead.
```

**Comment by thesunny** [comment]
I think this is just a matter of taking this line:  
  
[https://github.com/ianstormtaylor/slate/blob/623960a7d14103b8cebe667019b4de5f8ad1fd61/packages/slate-react/src/components/editable.tsx#L521](https://github.com/ianstormtaylor/slate/blob/623960a7d14103b8cebe667019b4de5f8ad1fd61/packages/slate-react/src/components/editable.tsx#L521)

and this line:

[https://github.com/ianstormtaylor/slate/blob/623960a7d14103b8cebe667019b4de5f8ad1fd61/packages/slate-react/src/components/editable.tsx#L523](https://github.com/ianstormtaylor/slate/blob/623960a7d14103b8cebe667019b4de5f8ad1fd61/packages/slate-react/src/components/editable.tsx#L523)

and convert the boolean `false` to the string `"false"`.

Good first PR for anyone!

**Comment by macrozone** [comment]
this has not been released and affects server side rendering, is it possible to release that as soon as possible?

**Comment by TheSpyder** [comment]
It's just been released as 0.65 (see #4273).

**Comment by craigdallimore** [comment]
Thank you to everyone involved.
